modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,205 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer (ALB) Operator
+Users that wish to install the AWS Load Balancer (ALB) Operator can do so as long as the following prerequisites are met:
+
+.Prerequisites
+
+* An existing {product-title} cluster with BYO-VPC installed in STS mode
```

Comment:
I'm not sure if this doc just for BYO-VPC and STS cluster? If yes then that is OK. 
Because ALBO (AWS Load Balancer Operator) can be installed any OCP on AWS, but the installation steps are different for non-STS and STS cluster. The latter is more complicated.

Reply:
> I'm not sure if this doc just for BYO-VPC and STS cluster? If yes then that is OK. Because ALBO (AWS Load Balancer Operator) can be installed any OCP on AWS, but the installation steps are different for non-STS and STS cluster. The latter is more complicated.

Thanks @lihongan,
   Based on what I know (even with all the back and forth/time on this issue), I believe this doc is just for BYO-VPC STS clusters (unless something has changed I am not aware of). @bchandra-ocp, can you confirm?

Reply:
@lihongan In ROSA, we will support only for BYO-VPC and STS clusters. There will be no need to use CCOCTL, which is used in the OCP documentation? There were few more bugs and instructions that don't apply to ROSA. Refer to the XCMSTRAT parent jira for more details. 
Ideally, these must have been tested with ROSA, STS clusters before ALBO/OCP 4.13 GA release. Feel free to test with instructions or using OperatorHub and provide instructions.

Reply:
Then that's OK.
Actually we have tested ALBO on OSD_CCS, ROSA non-STS, ROSA STS, ROSA HCP. Just need to note the prerequisite before installing ALBO.
For BYOVPC, user need to ensure the proper tags are added to VPC and subnets.
For STS, user need to prepare the secrets manually.